src/core/server/core_usage_data/migrations.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { SavedObjectUnsanitizedDoc } from '../saved_objects';
+import { migrateTo7141 } from './migrations';
+import type { CoreUsageStats } from './types';
+
+const type = 'obj-type';
+const id = 'obj-id';
+
+describe('#migrateTo7141', () => {
+  it('Resets targeted counter fields and leaves others unchanged', () => {
+    const doc = {
+      type,
+      id,
+      attributes: {
+        foo: 'bar',
+        'apiCalls.savedObjectsImport.total': 10,
+      },
+    } as SavedObjectUnsanitizedDoc<CoreUsageStats>;
+
+    expect(migrateTo7141(doc)).toEqual({
+      type,
+      id,
+      attributes: {
+        foo: 'bar',
+        'apiCalls.savedObjectsImport.total': 0,
```

Comment:
Do we explicitly want to set all of these fields to `0` if they were previously `undefined`? I'm fine if the answer is "yes", I just want to better understand the intent.

Reply:
We don't _need_ to, but the fields will all be initialized to `0` when the usage stats object is retrieved anyway so it doesn't make a practical difference. I opted for simplified logic here :+1: